src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/ExternalReferenceItemEntry.cs

```diff
@@ -32,8 +34,19 @@ protected override object GetValueWorker(string keyName)
                     StandardTableKeyNames.Column => _reference.Span.Start.Character,
                     StandardTableKeyNames.ProjectName => _reference.ProjectName,
                     StandardTableKeyNames.Text => _reference.Text,
+                    StandardTableKeyNames.ItemOrigin => ComputeOrigin(_reference),
+                    StandardTableKeyNames.Repository => _reference.Repository,
                     _ => null,
                 };
+
+            private static ItemOrigin ComputeOrigin(ExternalReferenceItem reference)
+                => reference.Scope switch
+                {
+                    ExternalScope.Repository => ItemOrigin.IndexedInRepo,
+                    ExternalScope.Organization => ItemOrigin.IndexedInOrganization,
+                    ExternalScope.Global => ItemOrigin.IndexedInThirdParty,
```

Comment:
So do we have ExternalScope as an enum because we need that a different layer than ItemOrigin is available in?